src/utils/update/beneficial_owners_managing_officers_data_fetch.ts

```diff
@@ -58,21 +59,37 @@ export const retrieveBeneficialOwners = async (req: Request, appData: Applicatio
 };
 
 export const retrieveManagingOfficers = async (req: Request, appData: ApplicationData) => {
+
   const companyOfficers = await getCompanyOfficers(req, appData[EntityNumberKey] as string);
-  if (companyOfficers) {
-    for (const officer of (companyOfficers.items || [])) {
-      logger.info("Loaded officer " + officer.officerRole);
-      if (officer.resignedOn === undefined) {
-        if (officer.officerRole === "managing-officer") {
-          const managingOfficer = mapToManagingOfficer(officer);
-          logger.info("Loaded Managing Officer " + managingOfficer.id + " is " + managingOfficer.first_name + ", " + managingOfficer.last_name);
```

Comment:
please remove name from logs

Reply:
This is deleted but I'll do it in the code that has it.